src/404.html

```diff
@@ -0,0 +1,17 @@
+---
+title: Not Found
```

Comment:
I added a 404 page for when people try to visit urls that don't exist as well. Today, we show a Netlify branded 404 page.